dvc/repo/__init__.py

```diff
@@ -78,7 +78,8 @@ def __init__(self, root_dir=None):
 
         self.config = Config(self.dvc_dir)
 
-        self.scm = SCM(self.root_dir)
+        no_scm = self.config["core"].get("no_scm", False)
+        self.scm = SCM(root_dir, not no_scm, no_scm)
```

Comment:
Please don't lose `self.root_dir`, it is not the same as `root_dir`.
```suggestion
        self.scm = SCM(self.root_dir, not no_scm, no_scm)
```

Reply:
Also, do we need to pass `not no_scm, no_scm`?

Reply:
Fixed, that was unnecessary.